### DISCUSSION TITLE
Never forked a project before so I wanted to ask permission.

### DISCUSSION THREAD

**Discussion by FitzM** [davish/obsidian-full-calendar#467] [Feature Requests]
Hello, your plugin is amazing! I love it.

 I wanted to create a few features that will extend the functionality of it such as: being able to drag and drop, a "zoom in" feature if you have tasks schedule for a time, and a increasing in the size of the default displayed timeline/events.

 I'm thinking the best way for me to do it is to fork your plugin and add to it. I just wanted to ask your permission since my ideas will heavily rely on your plugin. I'll be sure to give credit to you and your plugin in anything I end up publishing.

(I'm quite a way away from understanding enough to develop a plugin that's worth publishing though.)

**Comment by Flecart** [comment]
it's MIT licensed, so you can.
